core/domain/domain.en.yml

```diff
@@ -835,3 +835,30 @@ responses:
 
   utter_daily_ci__early_opt_out__cancel_ci_recommendation:
     - text: If you notice any changes or get symptoms, feel free to check with me to reassess your symptoms
+
+  ## Daily check-in invalid ID
+
+  utter_daily_ci__invalid_id__invalid_link:
+    - text: I’m sorry but this link does not seem to be valid
+
+  utter_daily_ci__invalid_id__try_again:
+    - text: Please try again and make sure, if you copy the link in your browser, that you copy the exact same thing
+
+  utter_ask_daily_checkin__invalid_id__want_assessment:
+    - text: If that doesn’t work, would you like to assess your symptoms?
+      buttons:
+        - title: Yes
+          payload: "/affirm"
+        - title: No
+          payload: "/deny"
+
+  utter_acknowledge_ask_anything_else:
```

Comment:
This might seem like a weird naming, but I wanted to preserve the link between really similar messages i.e. this one and "utter_ask_anything_else"